tests/gold_tests/body_factory/c/src/custom204plugin.cc

```diff
@@ -0,0 +1,153 @@
+/** @file
```

Comment:
Upon further discussion with @persiaAziz and @SolidWallOfCode, we may want to move the plugin source to a new directory that would be designated for test-only plugins that are not necessarily good examples of writing plugins outside of testing certain ATS functionality.